trampoline/src/main/java/com/iluwatar/trampoline/Trampoline.java

```diff
@@ -0,0 +1,84 @@
+package com.iluwatar.trampoline;
+
+import java.util.stream.Stream;
+
+/**When get is called on the returned Trampoline, internally it will iterate calling ‘jump’
+ on the returned Trampoline as long as the concrete instance returned is More,
+ stopping once the returned instance is Done. Essential we convert looping via recursion into iteration,
+ the key enabling mechanism is the fact that Trampoline.more is a lazy operation.
+ Trampoline in cyclops-react extends java.util.Supplier. Calling Trampoline.more we are basically creating
+ a Supplier that defers the actual recursive call, and having defered the call we can move it outside of the recursive loop.
+ This means we can define algorithms recursively in Java but execute them iteratively.*/
```

Comment:
Please refer to methods properly in Javadoc e.g. {@link package.class#member label}. What is cyclops-react?

Reply:
cyclops-react is a library for additions java8 streams.
done.